.github/workflows/ci-build.yml

```diff
@@ -168,6 +168,9 @@ jobs:
       with:
         lfs: true
 
+    - name: Set msg lib build type
+      run: echo "DEBUG=0" >> $GITHUB_ENV

```

Comment:
refer to [this link](https://docs.github.com/en/actions/learn-github-actions/environment-variables)
```yaml
name: Greeting on variable day

on:
  workflow_dispatch

env:
  DAY_OF_WEEK: Monday

jobs:
  greeting_job:
    runs-on: ubuntu-latest
    env:
      Greeting: Hello
    steps:
      - name: "Say Hello Mona it's Monday"
        run: echo "$Greeting $First_Name. Today is $DAY_OF_WEEK!"
        env:
          First_Name: Mona
```